docs/concepts/data-views.asciidoc

```diff
@@ -1,45 +1,45 @@
-[[index-patterns]]
-=== Create an index pattern
+[[data-views]]
```

Comment:
I think we should try to keep existing links working.
AFAIK we use this file to keep old linking working and to redirect to new URL: `docs/redirects.asciidoc`